### PR TITLE
Add a benchmark for CFStringInlineBuffer on bridged Strings

### PR DESCRIPTION
**Pull request by Catfish-Man**

### CODE REVIEW

**Comment by Catfish-Man** [comment]
@swift-ci please benchmark

**Comment by Catfish-Man** [comment]
@swift-ci please benchmark

**Comment by Catfish-Man** [comment]
@swift-ci please benchmark

**Comment by Catfish-Man** [comment]
@swift-ci please benchmark

File: benchmark/single-source/NSStringConversion.swift [hunk 1]
```diff
@@ -227,4 +241,31 @@ public func run_NSStringConversion_longNonASCII_rebridgeMutable(_ n: Int) {
...
+  buffer.deallocate()
+}
+
+public func run_NSStringConversion_inlineBuffer_ASCII(_ n: Int) {
+  let buffer = UnsafeMutablePointer<CFStringInlineBuffer>.allocate(capacity: 1)
+  for _ in 0 ..< n {
+    _iterateWithInlineBuffer(string: mutableTest, into: buffer)
+  }
+  buffer.deallocate()
+}
```
Comment (lorentey) [inline benchmark/single-source/NSStringConversion.swift:269]: This looks the same as `run_NSStringConversion_inlineBuffer_UTF8` -- any reason not to just share the same function in both benchmarks?

**Review by lorentey — APPROVED** [review]

File: benchmark/single-source/NSStringConversion.swift (hunk 1 above)
Comment (Catfish-Man) [reply benchmark/single-source/NSStringConversion.swift:269]: oh and just change the setup string? good point

**Review by glessard — APPROVED** [review]
LGTM!

**Comment by Catfish-Man** [comment]
@swift-ci please smoke test and merge

**Pull request merged**